CHANGELOG.md

```diff
@@ -7,7 +7,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-### Changed
+## Version 2.0.0 - Feb 24, 2020
+
+### rippled support
+
+This release adds support for the new rippled protocol buffers and adds browser compatibility support.
+
+New Protocol Buffers from rippled
+rippled implemented protocol buffer support natively, which uses a new and incompatible set of protocol buffers. The implementation in rippled was completed in: ripple/rippled#3159. Xpring4j is now capable of using these protocol buffers. This change is opt-in and non-breaking.
+
+To switch to these protocol buffers, pass true to the useNewProtocolBuffer parameter in XpringClient's constructor. The default for this field is false. The remote rippled node must have gRPC enabled.
+
+### Additional fixes
```

Comment:
```suggestion
### Additional Fixes
```
?

Not sure about the casing. `### rippled support` is all lowercase, but `### Breaking Changes` below is capitalized.